src/MongoBlock.ts

```diff
@@ -19,11 +19,17 @@ export class MongoBlock implements Block {
     const eosActions = []
     let currentTx = ""
     let actionIndex = 0
+    let currentActionDigest = ""
     for (const rawAction of rawActions) {
-      if (rawAction.trx_id != currentTx) {
+      if (rawAction.trx_id !== currentTx) {
```

Comment:
is there ever a time when action traces won't be sequential?  Or is it guaranteed they'll always be in sequence?

Reply:
This detects when actions transition from one transaction to another. They way they're ordered is grouped, so when we see the tx_id change, we know we're in a new transaction.

Reply:
To answer your question... this currently depends on insertion order, I believe. But, insertion order will always be by global action sequence, due to how the MongoDB plugin needs to operate.